opal/mca/memory/base/memory_base_open.c

```diff
@@ -65,31 +69,64 @@ static opal_memory_base_component_2_0_0_t empty_component = {
  */
 opal_memory_base_component_2_0_0_t *opal_memory = &empty_component;
 
+#if MEMORY_LINUX_PTMALLOC2
+#include "opal/mca/memory/linux/memory_linux.h"
+#endif
 
+void opal_memory_base_malloc_init_hook (void)
+{
+#if MEMORY_LINUX_PTMALLOC2
```

Comment:
@hjelmn Tells me that there's no component selected at this point.  Grumble.  Well, it's not a regression, so I guess we'll let this go for v2.0.0...